freqtrade/optimize/hyperopt.py

```diff
@@ -280,7 +280,7 @@ def generate_optimizer(self, _params) -> Dict:
             {
                 'stake_amount': self.config['stake_amount'],
                 'processed': processed,
-                'realistic': self.config.get('realistic_simulation', False),
+                'position_stacking': self.config.get('position_stacking', False),
             }
```

Comment:
For hyperopt I think it should default to true, it rarely makes sense running hyperopting for "just the first found trades".